### ISSUE TITLE
leading minus sign if the value is zero

### ISSUE THREAD

**Issue by rocthem** [fcambus/ansiweather#152]
Is it possible to remove the leading minus sign if the temperature value is zero?
See also attachment.
![2023-11-22_22-02](https://github.com/fcambus/ansiweather/assets/45518614/e8a9c126-e141-45d1-9846-99c1fde44cdf)

**Comment by fcambus** [comment]
No, see my [answer](https://github.com/fcambus/ansiweather/pull/153#issuecomment-3765382061) in PR #153 for the rationale.
